packages/@aws-cdk/aws-ecs/lib/firelens-log-router.ts

```diff
@@ -109,14 +110,22 @@ export interface FirelensLogRouterDefinitionOptions extends ContainerDefinitionO
 function renderFirelensConfig(firelensConfig: FirelensConfig): CfnTaskDefinition.FirelensConfigurationProperty {
   if (!firelensConfig.options) {
     return { type: firelensConfig.type };
+  } else if (firelensConfig.options.configFileValue === undefined) {
+    // config file options must be set together and are optional
+    return {
+      type: firelensConfig.type,
+      options: {
+        'enable-ecs-log-metadata': firelensConfig.options.enableECSLogMetadata ? 'true' : 'false',
+      },
+    };
   } else {
     // firelensConfig.options.configFileType has been filled with s3 or file type in constructor.
     return {
       type: firelensConfig.type,
       options: {
         'enable-ecs-log-metadata': firelensConfig.options.enableECSLogMetadata ? 'true' : 'false',
         'config-file-type': firelensConfig.options.configFileType!,
-        'config-file-value': firelensConfig.options.configFileValue,
+        'config-file-value': firelensConfig.options.configFileValue!,
```

Comment:
```suggestion
        'config-file-value': firelensConfig.options.configFileValue,
```

can you double check this? I think typescript should know that `configFileValue` is defined because of the `else if` condition above this.